include/z64player.h

```diff
@@ -930,4 +930,57 @@ typedef struct Player {
     /* 0x0A88 */ Vec3f unk_A88; // previous body part 0 position
 } Player; // size = 0xA94
 
+void Player_SetBootData(struct PlayState* play, Player* this);
```

Comment:
Think a small note like `// z_player_lib public functions` or something along those lines could be good